deployment/README.md

```diff
@@ -1,73 +1,148 @@
 
-### Amazon Machine Images (AMIs)
+ - Install [Packer]()
 
-In order to generate AMIs for the leader and followers, use the following `make` targets:
+## Create IAM Roles (optional)
 
-```bash
-$ make leader-ami
-$ make follower-ami
+Two IAM roles are required for the stack. These IAM roles are assigned to the `mesos-leader` and `mesos-follower` instances and grant access to S3. Assuming your data to be ingested is on S3, the profiles would include access to S3.
+
+Example `MesosLeaderInstanceProfile` and `MesosFollowerInstanceProfile`:
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "s3:*",
+      "Resource": "*"
+    }
+  ]
+}
 ```
 
-### CloudFormation (via Troposphere)
+If your data does not reside on S3 or needs access to other AWS resources your policy may need to be modified. If a profile is not provided in the `geotrellis-cluster.config` then your instances will not be launched with IAM profiles attached to them.
+
+
+## Create/Download EC2 KeyPair
 
-After at least one AMI of each type exists, use the following command to generate all of the CloudFormation templates:
+The EC2 KeyPair is used to authenticate SSH connections to the instances from a local machine. Creating this Key is necessary to launch an instance; however, if you already have a Key you can simply re-use that. Note, keys are tied to regions so if you want to launch a stack in a different region, you need to create a key for that region.
+
+## Configure AWS Profile using the AWS CLI
+
+Using the AWS CLI, create an AWS profile.
 
 ```bash
-$ make build
+$ aws configure
 ```
 
-#### Launch the AWS Virtual Private Cloud (VPC)
+You will be prompted to enter your AWS access keys,  AWS access key IDs, and default region. These credentials will be used to authenticate calls to the API when using Boto, Packer, and the AWS CLI.
+
+## Edit Configuration File
 
-Use the following command to create the VPC stack:
+A configuration file is required to launch the stack. An example file is available in this directory at `geotrellis-cluster.cfg.example`.
 
 ```
-$ make vpc-stack
+[CONFIG NAME]
+IPAccess: '<IP to allow SSH Access> (e.g. 216.158.51.82/32)'
+KeyName: '<EC2 Key to authenticate with SSH>'
+MesosFollowerAMI: '<AMI ID of Mesos Follower (optional -- will be found automatically if not provided)>'
+MesosFollowerInstanceProfile: '<ARN of Mesos Follow Instance Profile (optional -- may not be necessary)>'
+MesosFollowerInstanceType: '<Mesos Follow Instance Type>'
+MesosFollowerSpotPrice: '<Spot Price to use Mesos Follower Instances> (optional)>'
+MesosLeaderAMI: '<AMI ID of Mesos Leader (optional -- will be found automatically if not provided)>'
+MesosLeaderInstanceProfile: '<ARN of Mesos Leader Instance Profile (optional -- may not be necessary)>'
+MesosLeaderInstanceType: '<Mesos Leader Instance Type>'
+MesosNotificationARN: ''
+Region: '<AWS Region to Launch Stack>'
+StackType: '<Type of Stack to launch (e.g. accumulo)>'
```

Comment:
I think `PrivateHostedZoneName` may be missing from here and the example configuration file.